app/messages.pot

```diff
@@ -0,0 +1,22 @@
+# Translations template for PROJECT.
```

Comment:
Do you want to remove this file? I don't think we're keeping the "Hello world" string around

Reply:
I can remove until we have actual translations. Same as below

Reply:
yeah i'd remove. otherwise this lgtm.